src/ifcblenderexport/blenderbim/bim/prop.py

```diff
@@ -1716,7 +1729,7 @@ class BIMMeshProperties(PropertyGroup):
     is_swept_solid: BoolProperty(name="Is Swept Solid")
     swept_solids: CollectionProperty(name="Swept Solids", type=SweptSolid)
     is_parametric: BoolProperty(name="Is Parametric", default=False)
-    presentation_layer: StringProperty(name="Presentation Layer")
+    presentation_layer: PointerProperty(name="Presentation Layer", type=PresentationLayer)
```

Comment:
Removed for now, as you don't use this, instead you use the one assigned to BIMObjectProperties.

However, I _think_ it is more correct for it to belong here, because layers in IFC are actually assigned to representation items (i.e. Meshes in the Blender world), _not_ to the object itself. If you store it on the object, there is currently no way to distinguish the theoretical situation where an object has two representations, both assigned to two different layers.

This can fix this in the future.